src/entities/county/county.module.ts

```diff
@@ -0,0 +1,13 @@
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { County } from './county.entity';
+import { CountyController } from './county.controller';
+import { CountyService } from './county.service';
+import { CountyElectionResultModule } from '../county-election-result/county-election-result.module';
+
+@Module({
+  imports: [TypeOrmModule.forFeature([County]), CountyElectionResultModule],
```

Comment:
Please, don't mix `CountyElectionResultModule` with `Couty`

Reply:
Are you about imports on 6 and 9 lines? It's necessary to be able to inject `CountyElectionResultService` in `CountyService` constructor. 
For now I've implemented `/counties/:id/election-results` endpoint in the same controller (`CountyController`), near `/counties` and `/counties/:id`. Thus, all endpoints related to the county are located in one file.
If it's a bad decision and it generates unwanted dependencies, I can create controller for `CountyElectionResult` and put `/counties/:id/election-results` endpoint there.

Reply:
Ok. I've checked the API structure. I agree with you.